### ISSUE TITLE
Deploy a keyboard navigable version of this sim

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/friction#47]
We are ready to proceed with a minor release off of master with keyboard navigation for this sim. With the recent chipper 2.0 work, we grabbed a stable SHA of chipper to use with new RC's. Here it is:

  "chipper": {
    "sha": "6f8b4c139ef54b626df1ac052874826d4b646cc0",
    "branch": "master"
  },
@zepumph volunteered to prepare this RC, many thanks and assigning to him.

**Comment by jessegreenberg** [comment]
@phet-steele just showed me a multi touch issue while interacting with both a keyboard and a mouse, deployment should be on-hold until those are generally resolved.

**Comment by jessegreenberg** [comment]
Blocking issue: https://github.com/phetsims/scenery/issues/713

**Comment by jessegreenberg** [comment]
While we still want to bring in #49, phetsims/scenery#713 is sufficiently fixed to no longer be a blocking issue.

**Comment by zepumph** [comment]
https://github.com/phetsims/friction/issues/49 is the only issue left for this.

**Comment by zepumph** [comment]
We are ready for this.

**Comment by zepumph** [comment]
https://github.com/phetsims/QA/issues/75 in the queue. On hold here

**Comment by jessegreenberg** [comment]
This sim was deployed with keyboard navigation, this issue can be closed.
